modules/cert-manager-install-console.adoc

```diff
@@ -24,12 +24,23 @@ You can use the web console to install the {cert-manager-operator}.
 . Select the *{cert-manager-operator}* and click *Install*.
 
 . On the *Install Operator* page:
-.. The *Update channel* is set to *tech-preview*, which installs the latest Technology Preview release of the {cert-manager-operator}.
-.. The *Installation Mode* is set to *All namespaces on the cluster (default)*. This mode installs the Operator in the Operator-recommended `openshift-cert-manager-operator` namespace to watch and be made available to all namespaces in the cluster.
-.. Choose the *Installed Namespace* for the Operator. The default Operator recommended namespace is `openshift-cert-manager-operator`.
+.. The *Update channel* is set to *stable-v1*, which installs the latest stable release of the {cert-manager-operator}.
 +
-If the `openshift-cert-manager-operator` namespace does not exist, it is created for you.
-.. Click the *Enable Operator recommended cluster monitoring on the Namespace* checkbox to enable cluster monitoring for the Operator.
+// TODO: Right now I see "alpha", "stable-v1", "stable-v1.10", and tech-preview. Need to see if all of these will be available at GA or not, and if any shown should be avoided / descriptions provided
+.. The *Installation Mode* is set to *A specific namespace on the cluster*. This mode installs the Operator in the Operator-recommended `cert-manager-operator` namespace to watch and be made available to all namespaces in the cluster.
++
+// TODO: See if above description needs adjusted. Maybe delete the part about installing in the ns, because that is covered later
+.. Choose the *Installed Namespace* for the Operator. The default Operator recommended namespace is `cert-manager-operator`.
++
+// TODO: Confirm that the namespace was purposefully changed. Also check on the OLM files that are using "openshift-cert-manager-operator" to see if that is affected (Alex Greene)
```

Comment:
Hi @awgreene! The namespace that the cert-manager operator is being installed to is changing with our GA release. 

I saw several references to the old namespace of "openshift-cert-manager-operator" on this platform operators page: https://docs.openshift.com/container-platform/4.12/operators/admin/olm-managing-po.html

Two references are as a namespace, and the other is some sort of list of supported operators (but i'm not sure where this value comes from). 

Do you know if any of those references should also be changed with this namespace name change?

Reply:
@bergerhoffer "openshift-cert-manager-operator" is the package name of the Operator. See https://docs.openshift.com/container-platform/4.12/operators/understanding/olm-packaging-format.html#olm-file-based-catalogs_olm-packaging-format about the "name" within "olm.package". You can see:
$ oc get packagemanifest -n openshift-marketplace -o yaml
```
    defaultChannel: stable-v1
    packageName: openshift-cert-manager-operator
```
CC @thejasn @swghosh .

Reply:
@xingxingxia Thanks! So that means that the OLM docs are okay? We don't even need to update those namespace references in the OLM docs?

Reply:
1 For occurrences that reference "openshift-cert-manager-operator" as package name, need not update.
2 For occurrences that reference "openshift-cert-manager-operator" as namespace name, we should update for which seems you are already done.